README.md

```diff
@@ -52,55 +55,98 @@ accumulated remain preserved and accessible over time.
+- `Expense` and `Category` are basic object types we have for conveying information that are not separable. 
+For instance, an expense called "Walmart" would have no meaning unless we know the specific amount of money spent 
+shopping in Walmart. These classes belong to the Entity layer, but are passed around in both the `use_case` and `views` 
+packages, which is a Clean Architecture violation. We could remove the imports to these entities, and change the output
+data to `Object[][]` type in the `UpdateViewUseCaseInteractor` to accommodate Java Swing requirements. 
+However, we thought it is not adhering to SOLID principle from the point that `Object[][]` may not be the type required 
+by other view libraries that could be imported, which is in contrary to the Open/Closed Principle.
+- `SessionStorage` is an object type used to hold all information contained in the user's session. Every time a use case 
+is called, the interactor of that use case needs to access the information within the `SessionStorage` object. Thus, it 
 needs to be passed throughout all layers of our program, despite being a violation of Clean Architecture.
 
 ### SOLID Principles
 - **Single Responsibility Principle:** All classes are acted upon by a single actor and adhere to this principle.
-- **Open-Closed Principle:** Since our classes with main functionality implement IB and OB interfaces, the user is able 
-to modify any interactor or presenter as they see fit. For example, if the user wanted to implement another view that 
-generates statistical data with all months they could create another interactor to do that.
-- **Liskov Substitution Principle:** All of our IB and OB objects adhere to this principle. In order to adhere to the 
-DIP, the interactors and presenters are cast to the IB and OB, respectively. This demonstrates the LSP, as any class 
-that implements these interfaces can be used. Another example is the LoadMonthMenuVB, which 
+- **Open-Closed Principle:** Since our classes with main functionality implement Input Boundary and Output Boundary
+interfaces, the user is able to modify any interactor or presenter as they see fit. For example, if the user wanted to 
+implement another view that generates statistical data with all months they could create another interactor to do that.
+- **Liskov Substitution Principle:** All of our Input Boundary and Output Boundary objects adhere to this principle. 
+In order to adhere to the Dependency Inversion Principle, the interactors and presenters are cast to the Input Boundary 
+and Output Boundary, respectively. This demonstrates the LSP, as any class that implements these interfaces can be used.
 - **Interface Segregation Principle:** All of our interfaces adhere to this rule, as they are all small and contain only
 the necessary methods needed.
 - **Dependency Inversion Principle:** All classes have a boundary between every layer in Clean Architecture, to create 
 the dependency inversion.
 
 ### Design Patterns
```

Comment:
We should include the MVC pattern since our TA pointed it out in our rubric!

Reply:
Oooh, yep. I forgot that one also counts as a design pattern

Reply:
Does anyone remember why we ended choosing MVC over MVP? Cause I'd like to add our reasoning, but I don't think I remember our discussions about it

Reply:
Hmm I remember it was Yin's idea, but I am pretty sure it was just because it was suggested in the slides. Maybe we could mention how it helps structure our code in a way that adheres to Clean Architecture?

Reply:
Ok. I just added a new commit that contains MVC. Could you check it to see if I missed something?

Reply:
Beautiful!